src/Administration/Resources/app/administration/src/app/component/form/sw-text-editor/sw-text-editor-toolbar-button/sw-text-editor-toolbar-button.html.twig

```diff
@@ -15,7 +15,7 @@
         {% endblock %}
 
         {% block sw_text_editor_toolbar_button_color %}
-            <div v-else-if="buttonConfig.type === 'foreColor'">
+            <div v-else-if="buttonConfig.type === 'foreColor'" title="">
```

Comment:
Care to explain why you add an empty title on some Divs?

Reply:
`Reset titles in submenu, to prevent displaying parent titles.`
More in detail? :-)

Reply:
Can `div` elements have titles? :)

Reply:
I have deleted all title="" from the code and the only change I can see is that the forecolor button get's a title :thinking:

Reply:
That's the point. Without this `title=""` all elements inside the container will get the title of parent.
![image](https://user-images.githubusercontent.com/135993/78117547-14741880-7406-11ea-8432-2e632bcda068.png)

Reply:
same in table:
![image](https://user-images.githubusercontent.com/135993/78117841-6cab1a80-7406-11ea-97cf-462e497d86e8.png)

Reply:
This did not happen in my case :sweat_smile: but I get your point :+1:

Reply:
Ahhh, I understand xD
I got this behaviour in Chromium (Gnome, Manjaro) and Firefox (Windows), but really not in Chrome on Windows. Nice to know.